lib/python/qmk/cli/pid.py

```diff
@@ -0,0 +1,204 @@
+"""Generate and apply a QMK PID
+
+Generates a PID and replaces the one in the provided config.h files if VID is 0x03A8
+"""
+
+import os
+import fileinput
+import tempfile
+import subprocess
+from pathlib import Path
+
+import re
+import hashlib
+import json
+
+from milc import cli
+
+VID_regex = re.compile(r"""^#define\s+VENDOR_ID\s+(0x[0-9a-f]{4})$""", flags=re.IGNORECASE)
+PID_regex = re.compile(r"""^#define\s+PRODUCT_ID\s+(0x[0-9a-f]{4})$""", flags=re.IGNORECASE)
```

Comment:
```suggestion
PID_regex = re.compile(r"^(#define\s+PRODUCT_ID\s+)(0x[0-9a-f]{4})$", flags=re.IGNORECASE)
```
Just a tiny change to the regex. Also, you don't need multi-line strings.
More info below.